src/styles/partials/_typography.scss

```diff
@@ -1,11 +1,11 @@
 body {
-	font-family: Arial, Helvetica, sans-serif;
+	font-family: 'Roboto', sans-serif;
```

Comment:
@hemantjadon are the fonts being fetched from the external link ? Because they are not being loaded in the site.